addon/mixins/scoped-query-cache.js

```diff
@@ -2,12 +2,15 @@ import Ember from 'ember';
 import sortedObject from 'ember-scoped-query-cache/utils/sorted-object';
 const { assert } = Ember;
 
+const CACHE_DECAY_TIMEOUT = 5000;
+
 export default Ember.Mixin.create({
```

Comment:
If that's the case `cache` + `decayId` are likely too vaguely named, they can very easily collide with existing properties on the object it is being mixed into.